packages/python-packages/doc-warden/warden/version.py

```diff
@@ -1 +1 @@
-VERSION = '0.7.1'
+VERSION = '0.7.2'
```

Comment:
Dozie adding a new required parameter is a breaking change, not a patch version update. 

IMO this is a major version bump and I commented as such on the original PR IIRC. Did we decide not to do the major version bump for a reason?

Reply:
What I did was I made the parameter optional, so if it is not passed it defaults to the scan_directory
https://github.com/Azure/azure-sdk-tools/blob/0dacb3729dba0a93986a79e67939822caeb2a194/packages/python-packages/doc-warden/warden/WardenConfiguration.py#L52